pkg/k8s/deployments/translate.go

```diff
@@ -427,9 +439,59 @@ func TranslateOktetoSyncSecret(spec *apiv1.PodSpec, name string) {
 		Name: oktetoSyncSecretVolume,
 		VolumeSource: apiv1.VolumeSource{
 			Secret: &apiv1.SecretVolumeSource{
-				SecretName: fmt.Sprintf(oktetoSyncSecretTemplate, name),
+				SecretName: fmt.Sprintf(oktetoSecretTemplate, name),
+				Items: []apiv1.KeyToPath{
```

Comment:
is there a risk that these get mixed with the dev secrets if they have the same name?

Reply:
Good catch, I will add a prefix to the dev secrets ones to avoid collisions